inlong-manager/manager-dao/src/main/resources/mappers/InlongClusterEntityMapper.xml

```diff
@@ -198,4 +198,19 @@
         from inlong_cluster
         where id = #{id,jdbcType=INTEGER}
     </delete>
+
+    <select id="selectAllClusters"
+            resultType="org.apache.inlong.manager.common.pojo.sortstandalone.SortSourceClusterInfo">
+        select name,
+               type,
+               cluster_tags as cluster_tag,
```

Comment:
The `cluster_tags` may have more than one tag, they are separated by the commas(,), do you need to get one of them?

Reply:
has renamed to cluster_tags